src/plugins/mf_priority.cpp

```diff
@@ -914,6 +914,36 @@ static int depend_cb (flux_plugin_t *p,
 }
 
 
+static int run_cb (flux_plugin_t *p,
+                      const char *topic,
+                      flux_plugin_arg_t *args,
+                      void *data)
```

Comment:
looks like some whitespace errors here?

Reply:
Ah, yes, thanks. I always forget to adjust the whitespace for these function headers when I copy/paste the callbacks. 🤦 Just pushed up a change to fix this.